helper/testhelpers/postgresql/postgresqlhelper.go

```diff
@@ -12,44 +12,78 @@ import (
 )
 
 func PrepareTestContainer(t *testing.T, version string) (func(), string) {
-	return prepareTestContainer(t, version, "secret", "database")
+	env := []string{
+		"POSTGRES_PASSWORD=secret",
+		"POSTGRES_DB=database",
+	}
+
+	_, cleanup, url, _ := prepareTestContainer(t, "postgres", "postgres", version, "secret", "database", true, false, false, env)
+
+	return cleanup, url
 }
 
 func PrepareTestContainerWithPassword(t *testing.T, version, password string) (func(), string) {
-	return prepareTestContainer(t, version, password, "database")
+	env := []string{
+		"POSTGRES_PASSWORD=" + password,
+		"POSTGRES_DB=database",
+	}
+
+	_, cleanup, url, _ := prepareTestContainer(t, "postgres", "postgres", version, password, "database", true, false, false, env)
+
+	return cleanup, url
+}
+
+func PrepareTestContainerRepmgr(t *testing.T, name, version string, envVars []string) (*docker.Runner, func(), string, string) {
```

Comment:
Any good reason to not follow a similar patten by defining the env like the functions above? Looks like the envVars are coming from `testPostgreSQL_Repmgr_Container()` for this one.

Reply:
The test defines slightly different environment variables for the two different containers in the Repmgr test, which is why I parameterized it. But I can move the hardcoded ones here. Let me know if you think that looks better in the latest commit.

Reply:
LGTM